specification/powerbiprivatelinks/resource-manager/Microsoft.PowerBI/stable/2020-06-01/powerbiprivatelinks.json

```diff
@@ -0,0 +1,1627 @@
+        },
+        "resource": {
+          "type": "string",
+          "description": "The localized friendly form of the resource type related to this action/operation. This form should match the public documentation for the resource provider. Use Title Casing. For examples, refer to the \"name\" section."
+        },
+        "operation": {
+          "type": "string",
+          "description": "The localized friendly name for the operation as shown to the user. This name should be concise (to fit in drop downs), but clear (self-documenting). Use Title Casing and include the entity/resource to which it applies."
+        },
+        "description": {
+          "type": "string",
+          "description": "The localized friendly description for the operation as shown to the user. This description should be thorough, yet concise. It will be used in tool-tips and detailed views."
+        },
+        "origin": {
+          "type": "string",
+          "description": "The intended executor of the operation; governs the display of the operation in the RBAC UX and the audit logs UX. Default value is 'user,system'"
+        }
+      }
+    },
+    "Error": {
+      "title": "Error",
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Specifies the error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Specifies the error message."
+        },
+        "target": {
+          "type": "string",
+          "description": "Specifies the error target."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorDetail"
+          },
+          "description": "Specifies the error details."
+        }
+      }
+    },
+    "ErrorDetail": {
+      "title": "ErrorDetail",
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Specifies the error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Specifies the error message."
+        },
+        "target": {
+          "type": "string",
+          "description": "Specifies the error target."
+        }
+      }
+    },
+    "ErrorResponse": {
+      "type": "object",
+      "description": "Describes the format of Error response.",
+      "properties": {
+        "error": {
+          "description": "The error object",
+          "type": "object",
+          "properties": {
+            "code": {
+              "type": "string",
+              "description": "Error code"
+            },
+            "message": {
+              "type": "string",
+              "description": "Error message indicating why the operation failed."
+            }
+          }
+        }
+      }
+    },
+    "SystemData": {
```

Comment:
>SystemData [](http://example.com/codeflow?start=5&length=10)

Recommend using the definition from common-types.

Reply:
done

Reply:
updated